doc/source/whatsnew/v0.16.0.txt

```diff
@@ -22,6 +22,48 @@ New features
 
 .. _whatsnew_0160.api:
```

Comment:
your entry should be above this line (the link created here belongs to the title below your entry)